packages/react/src/components/Dropdown/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import ReactSelect from 'react-select';
+import Icon from '../Icon';
+import UiText from '../UiText';
+import { safeInvoke } from '../../utils';
+
+const customStyles = {
+  control: () => ({
+    border: 0,
+    backgroundColor: 'transparent',
+    minHeight: 'initial',
+    display: 'flex',
+    justifyContent: 'space-between',
+  }),
+  indicatorSeparator: provided => ({
+    ...provided,
+    display: 'none',
+  }),
+  indicatorContainer: provided => ({
+    ...provided,
+    marginRight: '1rem',
+    padding: 0,
+  }),
+  valueContainer: provided => ({
+    ...provided,
+    padding: 0,
+    marginLeft: '1rem',
+    minHeight: 38,
+  }),
+};
+
+const CustomDropdownIndicator = ({ innerRef, innerProps }) => (
+  <Icon
+    ref={innerRef}
+    {...innerProps}
+    icon={Icon.ICONS.IconChevronUpDown}
+    className='f-FormEl-icon'
+    size={Icon.SIZES.L}
+  />
+);
+
+const CustomSingleValue = ({ children }) => (
+  <UiText as='span' type={UiText.VARIANTS.content}>
```

Comment:
`Item`?

Reply:
WDYM?

Reply:
That is `UiText` because it represents the _selected_ value in the viewport of the select. Per design, even if the option itself is `Item` - meaning complex and rich, with arrows etc, only test is shown. This may be discussed with Juan further on if he thinks it's ok

Reply:
👍